src/components/issues/Card.tsx

```diff
@@ -107,6 +74,53 @@ const Card: FC<IssueCardProps> = ({ issue }) => {
                     Convert to task
                 </button>
             </div>
+            <p className={styles.card__issue_created__by}>
+                Opened on {created} by
```

Comment:
can we use `date` instead of created?

Reply:
please put clear variable name that is understandable

Reply:
Done